### ISSUE TITLE
User Guide link from home page is broken

### ISSUE THREAD

**Issue by desertnaut** [tensorly/tensorly#83]
Hi all,

The link to the User Guide from the **home** page (http://tensorly.org/stable/home.html) is broken, leading here:

http://tensorly.org/stable/user_guide.html

The link from the **index** page (http://tensorly.org/stable/index.html) is OK

**Comment by JeanKossaifi** [comment]
Yes, it is deployed automatically for the dev version by Travis and I'll push the fix for the stable version manually :)
